### PR TITLE
Joke drinks

### PR DESCRIPTION
**Pull request by JediKing112**
# About the pull request

Adds joke drinks and cocktails, which I will describe in an image. Needs testing and spriting - putting this up for testing.

# Explain why it's good for the game

Fun for MST and others

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl:
add: joke drinks like pilk
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry_reagents/drink.dm [hunk 1]
```diff
@@ -39,7 +39,91 @@
...
+	name = "FermisBrew"
+	id = "fermisbrew"
+	description = "A drink first made by physicist Enrico Fermi, who claimed it allowed him to see an alien world for a split second. Turns out he was right."
+	color = "3C5A3C" // rgb: 60, 90, 60
+
+/datum/reagent/drink/drinkofthegods
+	name = "DrinkOfTheGods"
+	id = "drinkofthegods"
+	description = "A fluid made from a number of other supposedly extremely powerful compounds. Many (of the few) who drink it never taste again, with the neurones for taste being completely destroyed upon the first sip. Said to have been god's creation on the 8th day, to chill out."
+	color = "#DCDCDC" // rgb: 220, 220, 220
```
Comment (carlarctg) [inline code/modules/reagents/chemistry_reagents/drink.dm:124]: for future reference you don't need to add the rgb in comments it's a pointless waste of time

Comment (JediKing112) [reply code/modules/reagents/chemistry_reagents/drink.dm:124]: but everybody else did it... i spent all that time..

**Comment by JediKing112** [comment]
im honestly tired of working on this, i dont even really want it to be merged that much, i just want the code to be fixed... and to know what i did wrong

**Comment by Zonespace27** [comment]
I know how much it sucks to have a PR stale out, so I'm just going to close this here.
The reason is that this doesn't feel like an appropriate or needed change considering the setting and level of seriousness CM is going for. Talking about a potential feature in the coder channels before coding it is a good way to get feedback or be told that the idea wouldn't work at all, which would avoid something like this happening.

**Pull request closed without merging**